src/main/scala/pureconfig/package.scala

```diff
@@ -111,4 +111,25 @@ package object pureconfig {
     rawConfig foreach { case (key, value) => printOutputStream.println(s"""$key="$value"""") }
     printOutputStream.close()
   }
+
+  /**
+   * Loads `files` in order, allowing values in later files to backstop missing values from prior, and converts them into a `Config`.
+   *
+   * This is a convenience method which enables having default configuration which backstops local configuration.
+   *
+   * Note: If an element of `files` references a file which doesn't exist or can't be read, it will silently be ignored.
+   *
+   * @param files Files ordered in decreasing priority containing part or all of a `Config`. Must not be empty.
+   */
+  def resolveFiles[Config: ConfigConvert](files: Traversable[java.io.File]): Try[Config] = {
```

Comment:
I wonder if the name of the method should be changed to match `loadConfig`. I get that `resolveFiles` is called like that because of how it works, but wouldn't something like `loadConfigFromFiles` be more consistent with the naming of `pureconfig`?